mmv1/products/cloudrun/api.yaml

```diff
@@ -554,6 +588,30 @@ objects:
                   explicitly specified, otherwise to an implementation-defined value.
                   The values of the map is string form of the 'quantity' k8s type:
                   https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/api/resource/quantity.go
+            - !ruby/object:Api::Type::Array
+              name: volumeMounts
+              min_version: beta
+              description: |-
+                Volume to mount into the container's filesystem.
+                Only supports SecretVolumeSources.
+              item_type: !ruby/object:Api::Type::NestedObject
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: mountPath
+                    required: true
+                    description: |-
+                      Path within the container at which the volume should be mounted.  Must
+                      not contain ':'.
+                  - !ruby/object:Api::Type::String
+                    name: name
+                    required: true
+                    description: |-
+                      This must match the Name of a Volume.
+                  - !ruby/object:Api::Type::Boolean
+                    name: readOnly
```

Comment:
Does this need to exist? If we don't send it does it get defaulted by the API? It doesn't seem like something a user wants to set if it can only have one possible value

Reply:
Removed.